src/ci/docker/scripts/rfl-build.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -euo pipefail
+
+LINUX_VERSION=c13320499ba0efd93174ef6462ae8a7a2933f6e7
+
+# Build rustc and rustdoc
+../x.py --stage 1 build library rustdoc
+
+# Install rustup so that we can use the built toolchain easily
+curl --proto '=https' --tlsv1.2 -sSf -o rustup.sh https://sh.rustup.rs
+sh rustup.sh -y --default-toolchain stable --profile minimal
```

Comment:
This step worries me. We're using a random stable here that's going to get bumped whenever this Docker image gets rebuilt, not at some "good" time. Can we pin this to a particular stable, similar to the pinned LINUX_VERSION above?

Or actually, maybe this stable isn't used for anything? If so, can we explicitly uninstall it?

Reply:
If it is not used, it could perhaps be a good idea to use a given stable one anyway for the step of building `bindgen-cli`, especially if we find it does give trouble.

If it is used, it would be nice to add a comment to clarify.

Reply:
Rustup is used for two things:
1) Allow easy installation of bindgen
2) Have an easy way of providing our built rustc to RfL

We don't really need the stable toolchain for anything, so I can change this to `--default-toolchain none`.